pkgs/development/libraries/cosmopolitan/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib, gcc9Stdenv, fetchFromGitHub, runCommand, cosmopolitan }:
+
+gcc9Stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+
+  src = fetchFromGitHub {
+    owner = "jart";
+    repo = "cosmopolitan";
+    rev = "94afa982c35d4322999b5591f6d41d5cfc3f19a5";
+    sha256 = "sha256-LaXkFMoArg5LUw4AdyhOqxzSJLdQCwHVBK2+lXVmxuI=";
+  };
+
+  postPatch = ''
+    patchShebangs build/
+    rm -r third_party/gcc
+  '';
+
+  dontConfigure = true;
+  dontFixup = true;
+  enableParallelBuilding = true;
+
+  preBuild = ''
+    makeFlagsArray=(SHELL=/bin/sh AS=${gcc9Stdenv.cc.targetPrefix}as CC=${gcc9Stdenv.cc.targetPrefix}gcc GCC=${gcc9Stdenv.cc.targetPrefix}gcc CXX=${gcc9Stdenv.cc.targetPrefix}g++ LD=${gcc9Stdenv.cc.targetPrefix}ld OBJCOPY=${gcc9Stdenv.cc.targetPrefix}objcopy "MKDIR=mkdir -p")
```

Comment:
This line is very long. Can we add some line breaks to try to keep a line length of ~100 chars?